src/lib/core/authorization/RedirectTransport.ts

```diff
@@ -0,0 +1,48 @@
+import PKCE from 'js-pkce';
+import type IConfig from 'js-pkce/dist/IConfig';
+import type IObject from 'js-pkce/dist/IObject';
+
+export class RedirectTransport {
+  #pkce: PKCE;
+
+  #params: IObject = {};
+
+  constructor(
+    options: IConfig & {
+      /**
+       * Additional parameters to be included in the query string of the authorization request.
+       */
+      params?: IObject;
+    },
+  ) {
+    const { params, ...config } = options;
+    this.#pkce = new PKCE({
+      ...config,
+    });
+    this.#params = {
+      ...params,
+    };
+  }
+
+  send() {
+    window.location.replace(this.#pkce.authorizeUrl(this.#params));
+  }
+
+  async getToken() {
```

Comment:
I could use a comment here to know when to use this method.

Reply:
Added some comments here ✅